hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -178,6 +178,15 @@ public enum ChecksumCombineMode {
       tags = { ConfigTag.CLIENT, ConfigTag.CRYPTO_COMPLIANCE })
   private int bytesPerChecksum = 16 * 1024;
 
+  // Client-side block chunk checksum cache config
+  // TODO: Re-evaluate if this need to be exposed to end-users at all, e.g. to avoid confusion.
+  @Config(key = "chunk.checksum.cache.enabled",
```

Comment:
IMO this should be the default behavior if incremental chunk list option is enabled.
Ok to simply drop this configuration property

Reply:
Agreed. I added this to facilitate testing (easier switching it on/off for comparison).